### PR TITLE
fix: [CDS-70093]: add more log to improve troubleshooting in container instance sync

### PR DESCRIPTION
**Pull request by rafaelmart1ns**
## Describe your changes
Adding more values on log to help us on troubleshooting.

## Checklist
- [ ] I've documented the changes in the PR description.
- [ ] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>

- Feature build: `trigger feature-build`
- 
  Specific builds
  - `trigger manager`
  - `trigger dms`
  - `trigger ng_manager`
  - `trigger cvng `
  - `trigger cmdlib`
  - `trigger template_svc`
  - `trigger events_fmwrk_monitor`
  - `trigger event_server`
  - `trigger change_data_capture`
  -  Trigger multiple builds together: For eg: `trigger dms, manager`
- Immutable delegate `trigger publish-delegate`
</details>

## [Latest PR Check Triggers](https://github.com/harness/harness-core/blob/develop/.github/pull_request_template.md)

<details>
  <summary>PR Check triggers</summary>
You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ut0, ut1`

- Compile: `trigger compile`
- CodeformatCheckstyle: `trigger checkstylecodeformat`
  - CodeFormat: `trigger codeformat`
  - Checkstyle: `trigger checkstyle`
- MessageMetadata: `trigger messagecheck`
- File-Permission-Check: `trigger checkpermission`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- Trigger CommonChecks: `trigger commonchecks`
- PMD: `trigger pmd`
- Copyright Check: `trigger copyrightcheck`
- Feature Name Check: `trigger featurenamecheck`
- UnitTests-ALL: `trigger uts`
- UnitTests-0: `trigger ut0`
- UnitTests-1: `trigger ut1`
- UnitTests-2: `trigger ut2`
- UnitTests-3: `trigger ut3`
- UnitTests-4: `trigger ut4`
- UnitTests-5: `trigger ut5`
- UnitTests-6: `trigger ut6`
- UnitTests-7: `trigger ut7`
- UnitTests-8: `trigger ut8`
- UnitTests-9: `trigger ut9`
- CodeBaseHash: `trigger codebasehash`
- CodeFormatCheckstyle: `trigger checkstylecodeformat`
- SonarScan: `trigger ss`
- GitLeaks: `trigger gitleaks`
- Trigger all Checks: `trigger smartchecks`
- Go Build: `trigger gobuild`
- Validate_Reviews: `trigger review`
- Module Dependency Check: `trigger mdc`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/50486)

### CODE REVIEW

**Comment by rafaelmart1ns** [comment]
trigger uts_JDK11

File: 400-rest/src/main/java/io/harness/perpetualtask/internal/PerpetualTaskRecordHandler.java [hunk 1]
```diff
@@ -165,7 +165,12 @@ public void assign(PerpetualTaskRecord taskRecord) {
         DelegateResponseData response = delegateService.executeTaskV2(validationTask);
 
         if (response instanceof ErrorNotifyResponseData) {
-          log.info("Perpetual validation task {} failed, unable to assign delegate.", validationTask.getUuid());
+          String errorMessage = ((ErrorNotifyResponseData) response).getErrorMessage();
+          String exception = ((ErrorNotifyResponseData) response).getException().toString();
+
+          log.info(
```
Comment (lasdox) [inline 400-rest/src/main/java/io/harness/perpetualtask/internal/PerpetualTaskRecordHandler.java:171]: shouldnt we change this to log.error so it adds support for stack trace?

Comment (rafaelmart1ns) [reply 400-rest/src/main/java/io/harness/perpetualtask/internal/PerpetualTaskRecordHandler.java:171]: This log already exists, I just added 2 new fields to it as it is the last log I accessed before the error.
@jennyjames21 @lasdox

Comment (jennyjames21) [reply 400-rest/src/main/java/io/harness/perpetualtask/internal/PerpetualTaskRecordHandler.java:171]: error message already present in the DB. 
you can print the exception but I suspect here the exception itself not coming from the delegate.

**Pull request closed without merging**